devtools/src/devtools.js

```diff
@@ -1,8 +1,12 @@
 import { options, Fragment, Component } from 'preact';
 
 export function initDevTools() {
-	if (typeof window != 'undefined' && window.__PREACT_DEVTOOLS__) {
-		window.__PREACT_DEVTOOLS__.attachPreact('10.4.1', options, {
+	const hook =
+		typeof window != 'undefined' &&
+		(window.__PREACT_DEVTOOLS__ || window.parent.__PREACT_DEVTOOLS__);
+
+	if (hook) {
+		hook.attachPreact('10.4.1', options, {
```

Comment:
kinda wondering if we could import the version from `package.json` and then inline during build, but not relevant for now